packages/flutter/test/material/tabs_test.dart

```diff
@@ -2216,6 +2217,36 @@ void main() {
 
   });
 
+  group('Tab feedback', () {
+    late FeedbackTester feedback;
+
+    setUp(() {
+      feedback = FeedbackTester();
+    });
+
+    tearDown(() {
+      feedback.dispose();
+    });
+
+    testWidgets('Tab enableFeedback can be disabled', (WidgetTester tester) async {
+      await tester.pumpWidget(
+        boilerplate(
+          child: const DefaultTabController(
+            length: 1,
+            child: TabBar(
+              tabs: <Tab>[
+                Tab(text: 'A',)
+              ],
+              enableFeedback: false,
+            )
+          )
+        )
+      );
+
+      expect(feedback.clickSoundCount, 0);
+    });
```

Comment:
This is not actually testing the click sound since you have not called the function to actually perform the tap on the widget but simply checking for a sound without tapping. Also this needs additional tests for 1.) enabled explicitly 2.) enabled by default. You may leave the enableFeedback feature out of this PR since it will be covered #69397 which is related to this issue and covers more widgets.

Reply:
@chinmoy12c , thanks for looking into this.
Sorry, I forgot to mark this test as work in progress (as I did for the other test).
I agree, the tests need to check for the default and when `enableFeedback` is set to false.
#69397 has a vague request. I would suggest resolving it with a no. of PRs rather than a single one.

Reply:
> @chinmoy12c , thanks for looking into this.
> Sorry, I forgot to mark this test as work in progress (as I did for the other test).
> I agree, the tests need to check for the default and when `enableFeedback` is set to false.
> #69397 has a vague request. I would suggest resolving it with a no. of PRs rather than a single one.

Yeah no problem :). Also from next time please comment on the issues that you're working on them to avoid conflicts, you see I was already working on this issue and I had mentioned that on the issue. I have the TabBar work finished already but since you have made the PR I would have to revert the work. So please check the comments section before submitting PRs.
Thank you :).

Reply:
@chinmoy12c , I was going through the issues and found that you had requested to work on many of them. This way, I would not have been able to work on any PR. I would suggest that rather than requesting to work on an issue, you can open a PR in draft mode, this way others can contribute more easily.
Thanks

Reply:
> @chinmoy12c , I was going through the issues and found that you had requested to work on many of them. This way, I would not have been able to work on any PR. I would suggest that rather than requesting to work on an issue, you can open a PR in draft mode, this way others can contribute more easily.
> Thanks

Okay that's fine. :)